app/routes/user/endorsements.tsx

```diff
@@ -215,49 +222,101 @@ export function EndorsementRequestCard({
 }) {
   const fetcher = useFetcher()
   const disabled = fetcher.state !== 'idle'
+  const [showMore, setShowMore] = useState(false)
+  const [isOverflowing, setIsOverflowing] = useState(true)
+  const ref = useRef<HTMLDivElement>(null)
+
+  function overflowing() {
+    if (ref.current)
+      setIsOverflowing(ref.current?.offsetWidth < ref.current?.scrollWidth)
+  }
+
+  useEffect(() => overflowing(), [])
 
   return (
     <fetcher.Form method="POST">
       {role === 'endorser' ? (
         <Grid row style={disabled ? { opacity: '50%' } : undefined}>
-          <div className="tablet:grid-col flex-fill">
+          <div className="tablet:grid-col flex-fill order-first">
             <div className="margin-y-0">
-              <strong>{endorsementRequest.requestorEmail}</strong>
+              <strong className="">{endorsementRequest.requestorEmail}</strong>
             </div>
+            {endorsementRequest.note && (
+              <div className="margin-top-0 margin-bottom-1">
+                Comments from the user:
+                {isOverflowing && (
+                  <Button
+                    type="button"
+                    unstyled
+                    aria-label="Show or hide the comments provided by the requestor"
+                    aria-expanded={showMore}
+                    onClick={() => setShowMore(!showMore)}
+                  >
+                    Show{' '}
+                    {showMore ? (
+                      <>
+                        less <Icon.ArrowDropUp />
+                      </>
+                    ) : (
+                      <>
+                        more <Icon.ArrowDropDown />
+                      </>
+                    )}
+                  </Button>
+                )}
+              </div>
+            )}
+          </div>
+          <div className="tablet:grid-col flex-auto order-last">
```

Comment:
We lost the `flex-center` prop here. Why?

Reply:
It squeezes the buttons down, 

With:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/d6bee1f7-57af-4ec7-a830-0bed425409c3)

Without:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/db287583-86d0-452c-84cf-8420892ec5e0)